### ISSUE TITLE
New users have [Object object] bio

### ISSUE THREAD

**Issue by jpaddison3** [LessWrong2/Lesswrong2#1987]
As reported by Habryka on May 16, I just confirmed on the most recent devel. Visible when hovering over username.

**Comment by Discordius** [comment]
Yeah, this is as a result of some inconsistencies in the way new-user callbacks are run. It should be relatively easy to fix by removing new user creation from the `makeEditableCallbacks` thing.
